core/action/ControlPoint.cc

```
@@ -127,19 +130,19 @@ void ControlPointRun::Merge(const G4Run* worker_run){
 ///
 ScoringMap& ControlPointRun::GetScoringCollection(const G4String& name){
     if (m_hashed_scoring_map.find(name) == m_hashed_scoring_map.end())
-        LOGSVC_ERROR("Couldn't find scoring collection in current run: {}",name)
+        // LOGSVC_ERROR("Couldn't find scoring collection in current run: {}",name)
     return m_hashed_scoring_map.at(name);
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)